pkg/daemon/ceph/osd/daemon.go

```diff
@@ -220,6 +220,15 @@ func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevi
 				} else if device.Name == desiredDevice.Name {
 					logger.Infof("%s found in the desired devices", device.Name)
 					matched = true
+				} else if strings.HasPrefix(desiredDevice.Name, "/dev/") {
+					devLinks := strings.Split(device.DevLinks, " ")
```

Comment:
This splitting feature is not mentioned in the docs. Why is this feature necessary? Shouldn't a single path be sufficient?

Reply:
This is splitting the output from udev, which reports multiple paths (one for each device symlink)